com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -608,6 +791,33 @@ private bool IsAnyRendererHasPostProcessingEnabled(UniversalRenderPipelineAsset
             return false;
         }
 
+        private static Texture2D LoadConsoleIcon(bool isError)
```

Comment:
I think this method should be moved to `CoreEditorUtils` and `m_ErrorIcon` and `m_WarningIcon` to `CoreEditorStyles`